modules/admin_manual/pages/maintenance/migrating.adoc

```diff
@@ -143,7 +142,7 @@ cd /var/www/owncloud/
 sudo -u www-data php occ maintenance:mode --on
 ----
 
-After that’s done, wait for 6-7 minutes and stop Apache:
+After that’s done, wait for some minutes and stop your web server, in this case Apache:
```

Comment:
s/wait for some minutes/wat a few minutes/g

Reply:
then 
s/wat/wait/
;)